app/build.gradle

```diff
@@ -0,0 +1,82 @@
+apply plugin: 'com.android.application'
+
+android {
+    compileSdkVersion 26
+    flavorDimensions "version"
+
+    signingConfigs {
+        releaseconfig {
+            keyAlias 'recipe'
+            keyPassword 'application'
+            storeFile file('/home/shubhank/AndroidStudioProjects/RecipeApplication1/app/build/outputs/apk/Staging/signedapk/staging-release.jks')
+            storePassword 'recipeapp'
+        }
+    }
+    defaultConfig {
+        applicationId "com.simpragma.recipe.recipeapplication1"
+        minSdkVersion 19
+        targetSdkVersion 26
+        versionCode 1
+        versionName "1.0"
+        testInstrumentationRunner "android.support.test.runner.AndroidJUnitRunner"
+    }
+    buildTypes {
+        release {
+            debuggable false
+            minifyEnabled true
+            shrinkResources true
+            proguardFiles getDefaultProguardFile('proguard-android.txt'), 'proguard-rules.pro'
+            signingConfig signingConfigs.releaseconfig
+        }
+        debug {
+            debuggable true
+            minifyEnabled false
+            shrinkResources false
+            proguardFiles getDefaultProguardFile('proguard-android.txt'), 'proguard-rules.pro'
+        }
+    }
+    productFlavors {
+        Staging {
+            //select the dimension of flavor
+            dimension "version"
+            //configure applicationId for app published to Play store
+            applicationId "com.simpragma.recipe.stage"
+            //Configure this flavor specific app name published in Play Store
+            //resValue "string", "flavored_app_nae.st", "Free Great App"
+            signingConfig signingConfigs.releaseconfig
+            targetSdkVersion 20
+        }
+        Production {
+            //select the dimension of flavor
+            dimension "version"
+            //configure applicationId for app published to Play store
+            applicationId "com.simpragma.recipe"
+            //Configure this flavor specific app name published in Play Store
+            //resValue "string", "flavored_app_name", "Free Great App"
+            minSdkVersion 21
```

Comment:
Why is this in flavor level? It should be in the app level

Reply:
I remove SDKversion.